tests/Reporter/GitlabReporterTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FriendsOfTwig\Twigcs\Tests\Reporter;
+
+use FriendsOfTwig\Twigcs\Reporter\GitlabReporter;
+use FriendsOfTwig\Twigcs\Validator\Violation;
+use JsonException;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Output\ConsoleOutput;
+
+class GitlabReporterTest extends TestCase
+{
+    public const EXPECTED_REPORT = <<<EOF
```

Comment:
@frankdekker 

This constant is used within a single test only. Can we in-line it into the test as a local variable?

Reply:
@localheinz I could, but i followed the format of JsonReporter and CheckStyleReporter. Personally I feel inlining the content will not improve the readability of the test. Still want the change?